src/models/Categories.tsx

```diff
@@ -0,0 +1,62 @@
+import { JSX } from 'preact/jsx-runtime'
+import Coin from 'icons/Coin'
+import ERC721ProofSection from 'components/proofs/ERC721ProofSection'
+import Email from 'icons/Email'
+import EmailProof from 'components/proofs/EmailProof'
+import EmailProofType from 'helpers/EmailProof'
+import Network from 'models/Network'
+import Nft from 'icons/Nft'
+import ProofSection from 'components/ProofSection'
+import ReadyEmailProof from 'components/proofs/ReadyEmailProof'
+
+export interface CategoriesComponentProps {
+  currentCategory: CategoriesTitles
+  setCategory: (category: CategoriesTitles) => void
+}
+
+interface Category {
+  [title: string]: {
+    icon: JSX.Element
+    contentToRender: (
+      account?: string,
+      emailProofsCompleted?: readonly EmailProofType[]
+    ) => JSX.Element | null
+    disabled?: boolean
+  }
+}
+
+export const categories: Category = {
```

Comment:
We can combine it with `data` from [this pr](https://github.com/BigWhaleLabs/seal-cred-frontend/pull/452)